Updates to reflect qualifying membership. (#177)
OfficeDev/office-dev-program-docs: docs/microsoft-365-developer-program-get-started.md

---
title: Request a Microsoft 365 developer sandbox subscription
description: Set up a Microsoft 365 developer subscription for building solutions independent of your production environment.
ms.date: 04/01/2019
ms.localizationpriority: high
---

# Request a Microsoft 365 developer sandbox subscription 

Qualifying Microsoft 365 Developer Program members can set up a Microsoft 365 E5 developer subscription for use in building solutions independent of your production environment. The Microsoft 365 E5 developer sandbox subscription includes 25 user licenses and lasts for 90 days. If you use your subscription for valid development activity, it will renew automatically. Commercial transactions, including purchasing paid services, are not currently supported.

> [!NOTE] 
> Before you request a developer subscription, you must first [join the Microsoft 365 Developer Program](microsoft-365-developer-program.md) directly or through Visual Studio Professional or Enterprise (if you're a subscriber). 

> If you qualify for a Microsoft 365 E5 subscription through the developer program, the subscription set up flow will start automatically after you join the program. If you choose not to set up your subscription right away, the option to set up a subscription will be available on your [Microsoft 365 Developer Program dashboard](https://developer.microsoft.com/en-us/microsoft-365/profile).

Reviewer: This is not accurate if we are removing the configurable sandbox. Recommend to remove.

Author: For now we aren't removing the configurable sandbox option but we will need to update this later when we are able to do that.

> If you don't automatically qualify for a developer subscription, after you join the program, you will see an option to provide additional information to validate that you are a qualified member. Submit the information and if you qualify, you will receive an email inviting you to set up your subscription within two weeks. If you don't receive an email, you don't currently qualify for a subscription.

Reviewer 2: ```suggestion
If you don't automatically qualify for a developer subscription, after you join the program, you'll see an option to provide additional information to validate that you're a qualified member. Submit the information and if you qualify, you'll receive an email inviting you to set up your subscription within two weeks. If you don't receive an email, you don't currently qualify for a subscription.
```
